### PR TITLE
Use sudo for elevated permissions while upgrading clusters using kubeadm

### PR DESCRIPTION
**Pull request by adityasamant25**
The pages for [Upgrading kubeadm clusters](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/) and [Upgrading Linux nodes](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/upgrading-linux-nodes/) have reference to `apt`, `apt-cache`, `apt-mark`, `apt-get`, `yum` and `kubeadm upgrade apply` commands. These commands need elevated permissions through `sudo` (if executed without root).

The PR adds the usage of `sudo` for these commands to ease the experience for the reader.

Regards,
Aditya

### CODE REVIEW

**Comment by T-Lakshmi** [comment]
I think its worth to mention sudo.
Looking good to me

Preview page:
[Upgrading Linux nodes](https://deploy-preview-44832--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/kubeadm/upgrading-linux-nodes/) | [Upgrading kubeadm clusters](https://deploy-preview-44832--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)

**Comment by adityasamant25** [comment]
> are these all the commands that need sudo on the pages? /lgtm

I have executed the steps on this page several times. I am pretty confident that all commands that need sudo have been taken care of in this PR.

**Comment by T-Lakshmi** [comment]
> we had a discussion before that the sudo requirement can be just a note on top of the doc instead of having sudo on every command.

Good suggestion,
I believe this approach is an efficient way of documenting.

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The upgrade workflow at high level is the following:
   they could be running CoreDNS Pods or other critical workloads. For more information see
   [Draining nodes](/docs/tasks/administer-cluster/safely-drain-node/).
 - The Kubernetes project recommends that you match your kubelet and kubeadm versions.
-  You can instead use an a version of kubelet that is older than kubeadm, provided it is within the
+  You can instead use a version of kubelet that is older than kubeadm, provided it is within the
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:46]: (nit)
I'd have put this fixup in a separate commit.

**Review by sftim — COMMENTED** [review]
Consider mentioning `sudo` in the prerequisites, and ideally tell people that alternative tools work fine.

(I like using PolicyKit locally, and sometimes use alternatives to `sudo` on servers).

Anyway:
/approve
